core/src/main/java/org/geysermc/geyser/network/netty/GeyserServer.java

```diff
@@ -199,11 +212,30 @@ private ServerBootstrap createBootstrap() {
         GeyserServerInitializer serverInitializer = new GeyserServerInitializer(this.geyser);
         playerGroup = serverInitializer.getEventLoopGroup();
         this.geyser.getLogger().debug("Setting MTU to " + this.geyser.getConfig().getMtu());
+
+        String geyserRakPacketLimit = System.getProperty("Geyser.RakPacketLimit");
+        int rakPacketLimit = geyserRakPacketLimit != null ? Integer.parseInt(geyserRakPacketLimit) : DEFAULT_PACKET_LIMIT;
```

Comment:
might be worth adding some exception catchers in case someone wrongly specifies values there